File: second_task/app/src/main/java/com/example/secondtask/main/model/TetaMenza.java
package com.example.secondtask.main.model;

import androidx.annotation.NonNull;

public class TetaMenza {

    private String first_name, last_name, workplace;
    private int age, OIB;

    public TetaMenza(String first_name, String last_name, String workplace, int age, int OIB) {
        this.first_name = first_name;
        this.last_name = last_name;
        this.workplace = workplace;
        this.age = age;
        this.OIB = OIB;
    }

    public String getFirst_name() {
        return first_name;
    }

    public String getLast_name() {
        return last_name;
    }

    public String getWorkplace() {
        return workplace;
    }

    public int getAge() {
        return age;
    }

    public int getOIB() {
        return OIB;
    }

    public void setFirst_name(String first_name) {
        this.first_name = first_name;
    }

    public void setLast_name(String last_name) {
        this.last_name = last_name;
    }

    public void setWorkplace(String workplace) {
        this.workplace = workplace;
    }

    public void setAge(int age) {
        this.age = age;
    }

    public void setOIB(int OIB) {
        this.OIB = OIB;
    }

    @NonNull
    @Override
    public String toString() {
        return first_name+" "+last_name+" "+age+" "+OIB+" "+workplace+"\n";
    }
}

File: second_task/app/src/main/java/com/example/secondtask/main/model/TetaMenza.kt
package com.example.secondtask.main.model

import com.example.secondtask.main.model.enums.Job_Position
import com.example.secondtask.main.model.enums.Workplace
import java.io.Serializable

class TetaMenza constructor(
        var first_name :String,
        var last_name : String,
        var workplace : Workplace,
        var age : Int,
        var OIB : Int,
        var gender : String,
        var job_position : Job_Position,
        var job_shift : String,
        var email : String,
        var address : String
):Serializable{

    override fun toString(): String {
        return first_name+" "+last_name+" "+workplace.toString()+" "+ age.toString()+" "+OIB.toString()+" "+gender+
                " "+job_position.toString()+" "+job_shift+" "+email+" "+address
    }
}